frontend/src/app/organization/organization-list/organization-list.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { OrganizationService } from '../organization.service';
+import { Organization } from '../organization.model';
+
+@Component({
+  selector: 'volontulo-organization-list',
+  templateUrl: './organization-list.component.html',
+  providers: [OrganizationService]
```

Comment:
Each `OrganizationService` provider will create new instance of `OrganizationService`. Hence, when used that way, you'll get the service instance on per-component basis. That's pretty often intended, but I don't think that's what you really want here, as you'll probably want to keep in there some state at some point (since apparently there's no redux-like global state here). That being said, I think it's better to provide this service via module, not via component.